src/charts/histogramBoxPlot/onResize.js

```diff
@@ -99,9 +155,9 @@ export default function onResize() {
           })
```

Comment:
I think the black lines are helpful now that we're moving to a light gray background.